.evergreen/garasign_dist_file.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Used by make_release.py.
+# See: https://docs.devprod.prod.corp.mongodb.com/release-tools-container-images/garasign/garasign_signing/
+
+set -o errexit
+set -o pipefail
+
+: "${1:?"missing dist_file as first argument"}"
+
+if ! command -v podman >/dev/null; then
```

Comment:
I (selfishly) request adding `docker` as a fallback:

```sh
declare launcher
if command -v podman >/dev/null; then
  launcher=podman
elif command -v docker >/dev/null; then
  launcher=docker
else
  echo "podman or docker is required to sign distribution tarball" 1>&2
fi
```

At present, I am unable to install `podman`:
```sh
podman: This software does not run on macOS versions older than Ventura.
Error: podman: An unsatisfied requirement failed this build.
```

I expect that is a short-term issue (I can upgrade), but this help me to release in the short-term.

Reply:
Added support for a `GARASIGN_LAUNCHER` env var that may be defined as a means to select an alternative to `podman`.